src/System.Windows.Forms/src/System/Windows/Forms/Controls/TabControl/TabRenderer.cs

```diff
@@ -144,7 +143,16 @@ public static void DrawTabPage(Graphics g, Rectangle bounds)
 
     internal static void DrawTabPage(IDeviceContext deviceContext, Rectangle bounds)
     {
-        InitializeRenderer(VisualStyleElement.Tab.Pane.Normal, 0);
+#pragma warning disable WFO5001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+        // Using DarkMode Theme Subclass.
+        // see https://learn.microsoft.com/en-us/windows/win32/controls/theme-subclasses.
```

Comment:
Links shouldn't have locale information
```suggestion
        // see https://learn.microsoft.com/windows/win32/controls/theme-subclasses.
```

Reply:
thanks for this information.